pkgs/unified_analytics/pubspec.yaml

```diff
@@ -4,7 +4,7 @@ description: >-
   to Google Analytics.
 # When updating this, keep the version consistent with the changelog and the
 # value in lib/src/constants.dart.
-version: 0.1.1
+version: 0.1.2
```

Comment:
If you don't intend to publish this you may want to do `0.1.2-dev` here; if you do intend to publish, lgtm.